.phpstorm.meta.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+namespace PHPSTORM_META;
```

Comment:
Shouldn't something be added to this file? Else delete it?

Reply:
Sure. Thanks. Missed it